lib/Mailer/Methods/ErrorMappers/MailPoetMapper.php

```diff
@@ -95,7 +95,7 @@ private function getSubscribersErrors($result_parsed, $subscribers) {
   private function getUnauthorizedEmailMessage($sender, $newsletter) {
     $email = $sender ? $sender['from_email'] : null;
     $message = '<p>';
-    $message .= sprintf(WPFunctions::get()->__('The MailPoet Sending Service did not send your latest email because the address <i>%s</i> is not yet authorized.', 'mailpoet'), $email ?: WPFunctions::get()->__('Unknown address'));
+    $message .= sprintf(WPFunctions::get()->__('The MailPoet Sending Service did not send your latest email because the address %s is not yet authorized.', 'mailpoet'), $email ?: '<i>' . WPFunctions::get()->__('Unknown address') . '</i>' );
```

Comment:
@ramiy The code before and after is not equivalent.

Before both the email address and the "Unknown address" strings would be italicized.
After the change, only the "Unknown address" string would be italicized, but the email address would be regular.

Could you please retain the original functionality?

Reply:
fixed